src/org/omegat/util/gui/UIDesignManager.java

```diff
@@ -477,6 +477,12 @@ private static void setDefaultColors() {
         Color hilite;
         if (isDarkTheme()) {
             hilite = UIManager.getColor("TextArea.background").brighter();  // NOI18N
+            // Hack for JDK GTKLookAndFeel bug.
+            // TextPane.background is always white but should be a text_background of GTK.
+            // List.background is as same color as text_background.
+            if (Platform.isLinux()) {
```

Comment:
I'm basically OK with this, but I wonder: what about other LAFs? Would this cause a regression in a LAF that sets `TextPane.background` more appropriately?

Instead of checking `isLinux()`, what about a check to see if `TextPane.background` is white?

Reply:
> Instead of checking isLinux(), what about a check to see if TextPane.background is white?

Oh, it is better way to check.